Settings-default.toml

```diff
@@ -146,13 +146,13 @@ anonymous_access = true
 
 # Settings for Geo Engine Pro
 [user]
-user_registration = true
+registration = true
```

Comment:
if you change stuff in here, you have to update the files in the server-deployment repo as well

Reply:
Nightly does not use it and other instances don't use this setting afaik.

Reply:
Actually, I don't know. Maybe start creating a changelog (again)?.
I could open up a PR for these changes as a draft, but we both know that we would forget this PR.